server/sonar-server/src/main/java/org/sonar/server/permission/ws/UsersAction.java

```diff
@@ -149,6 +153,7 @@ private static UsersWsResponse buildResponse(List<UserDto> users, List<UserPermi
         .setLogin(user.getLogin())
         .addAllPermissions(permissionsByUserId.get(user.getId()));
       setNullable(user.getEmail(), userResponse::setEmail);
```

Comment:
Existing email fields should be deprecated => Please update the JIRA ticket to say 'email' field is deprecated and add a changelog entry

Reply:
No, emails are still used and displayed on the page.

Reply:
ohhhh :(